ElectronClient/gui/EncryptionConfigScreen.jsx

```diff
@@ -116,6 +116,16 @@ class EncryptionConfigScreenComponent extends React.Component {
 
 			if (!answer) return;
 
+			if (!isEnabled) {
+				let confirmAnswer = '';
+				let confirmPasswordMessage = 'Re-enter your password to verify it.';
+				do {
+					confirmAnswer = await dialogs.prompt(_(confirmPasswordMessage), '', '', { type: 'password' });
+					confirmPasswordMessage = 'Passwords don\'t match!\nRe-enter your password to verify it.';
+				} while (confirmAnswer != answer);
```

Comment:
So if the user mistyped their first password, we force them to forever retry?

Reply:
That is what I meant with `What happens, if you cancel the dialog?`. ;-)

Nothing happens....

Reply:
I looked at the discussions but unfortunately using smalltalk prompts only allow one input from the user per time so adding two prompts require a lot of refactoring.

User can exit by clicking 'cancel'.

I realized the pain in forcing user to forever retry entering their passwords but what do you suggest?

Reply:
> User can exit by clicking 'cancel'.

Have you tried this? According to the code, the dialog pops up again when you click cancel.

Reply:
>  the dialog pops up again when you click cancel.

when you click cancel you get back to the encryption home page same as canceling the main "enter password" dialog, it keeps popping up just in case the user enters a wrong password.

Reply:
> when you click cancel you get back to the encryption home page 

No you wouldn't.

Reply:
![confirmpassword](https://user-images.githubusercontent.com/30902406/76914354-b225fe80-68c2-11ea-9f20-7e148432514d.gif)

this is the behavior, maybe the first gif didn't show the complete flow